docs/getting_started/video_tutorials/hyperparameter_optimization.md

```diff
@@ -18,44 +18,40 @@ title: Hyperparameter Optimization
 
 By now, we know that ClearML can easily capture our hyperparameters and scalars as part of the experiment tracking. We also know we can clone any task and change its hyperparameters so they’ll be injected into the original code at runtime. In the last video, we learnt how to make a remote machine execute this task automatically by using the agent. 
 
-Soooo…. Can we just clone a task like a 100 times, inject different hyperparameters in every clone, run the clones x_distributed_x on 10 agents and then sort the results based on a specific scalar?
+Soooo… Can we just clone a task like a 100 times, inject different hyperparameters in every clone, run the clones on 10 agents and then sort the results based on a specific scalar?
 
-Yeah, yeah we can, it's called hyperparameter optimization. And we can do all of this automatically too, no way you were going to clone and edit those 100 tasks yourself right?
+Yeah, yeah we can, it's called hyperparameter optimization. And we can do all of this automatically too! No way you were going to clone and edit those 100 tasks yourself, right?
 
 If you don’t know what Hyperparameter Optimization is yet, you can find a link to our blog post on the topic in the description below. But in its most basic form, hyperparameter optimization tries to optimize a certain output by changing a set of inputs. 
 
-Let’s say we’ve been working on this model here and we were tracking our experiments with it anyway. We can see we have some hyperparameters to work with in the hyperparameters tab of the webUI. They are logged by using the task.connect function in our code. These are our inputs. We also have a scaler called validation epoch_accuracy, that we want to get as high as possible. This is our output. We could also select to minimize the epoch_loss for example, that is something you can decide yourself.
+Let’s say we’ve been working on this model here, and we were tracking our experiments with it anyway. We can see we have some hyperparameters to work with in the hyperparameters tab of the webUI. They are logged by using the `task.connect` function in our code. These are our inputs. We also have a scaler called `validation/epoch_accuracy`, that we want to get as high as possible. This is our output. We could also select to minimize the `epoch_loss` for example, that is something you can decide yourself.
 
-For a simple optimizer like RandomSearch, we will simply run a bunch of random combinations of hyperparameters and sort the tasks based on the value of the output scalar. So the best performing task will be on top. But more complex optimizers like Optuna use bayesian optimization, which actually looks at this scalar to see how it just changes and tries to predict the best possible next set of hyperparameters to try out. So it’s not just a shot in the dark, pretty neat right?
```

Comment:
@pollfly Why did you take this out?

Reply:
It isn't in the video and I'm not sure whether it will be confusing or not. It depends whether the user is reading along with the video or separately.

Reply:
@ainoam I've decided to keep it out because it doesn't really fit in its original location (between 2 paragraphs about scalars)